### DISCUSSION TITLE
Alpine Nextcloud LXC and Nextcloud Hub

### DISCUSSION THREAD

**Discussion by ayoahha** [tteck/Proxmox#2708] [Help Needed] [answered]
Hello
I am trying to deploy alpine-nextcloud LXC and i see that the Hub version is still Hub 6.
Is there any possibility to update it to Hub 7 ?
Many thanks

Note : or even nextcloud 28... Is there any issue ?

**Comment by iodize6399** [comment]
I have a similar question.

Running the script inside the Nexcloud-alpine LXC does show an option to update v28, however, it does not do so.

![image](https://github.com/tteck/Proxmox/assets/4197707/b2adb2a9-6890-4209-8932-bd2ead26948b)
![image](https://github.com/tteck/Proxmox/assets/4197707/c0766112-e491-4b24-9586-d88a9aaf6b5a)

**Comment by iodize6399** [comment]
I would also like to know if updating LXC using `Proxmox VE LXC Updater` script cause any compatibility issues with a Nextcloud-Alpine LXC since it has its own script to update Nextcloud as shown above?

**Comment by tteck** [comment, marked as answer]
Versions depend on the Alpine branch, 3.18, 3.19 or edge.
https://pkgs.alpinelinux.org/packages?name=nextcloud&branch=v3.19&repo=&arch=&maintainer=

**Reply by ayoahha** [reply]
Thanks for the answer 
Is there any inconvenients to have edge version ?

**Reply by tteck** [reply]
Using the "edge" branch means you'll have access to bleeding-edge features and packages, but it also implies potential instability and bugs since the software is still undergoing development and testing.

**Reply by ayoahha** [reply]
OK, surely not a good idea !

May i ask you if performances are different from alpine CT and docker aio one ?
Or did you notice some differences

**Reply by tteck** [reply]
🤷🏻‍♂️  I don't use Nexcloud.
